src/client/test/components/home/index.test.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render, cleanup } from '@testing-library/react';
 
-import Home from 'components/home';
+import Home from 'client/components/home';
```

Comment:
This change makes me notice your folder structure.
Maybe it's better to move them one level upper? You might not want the `client` folder there and it sounds weird if it's inside a front-end like folder 🤔?  Maybe you'll add back-end server in the future? so I think current `src` is your `client` which means you might not want to create one level `client` inside a client.

For example, I guess you will want to make it like

```
├── client
│   ├── components
│   │   └── matchResults
│   │       └── index.js
│   ├── index.js
│   └── serviceWorker.js
└── server
    ├── analysis.js
    └── index.js
```

This is just an example. No need to follow it but you can think about why you want to create a `client` folder and how you'd like to organize this project.

Reply:
Yes, I do want to add `sever` folder in the future so I created `client` folder first. I'll keep thinking what is the better folder structure. Thanks!